.github/workflows/reindex-website-release.yml

```diff
@@ -1,17 +0,0 @@
-name: "Reindex website with Algolia Docsearch for Releases"
-on:
-  push:
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
-jobs:
-  reindex-website-release:
-    runs-on: ubuntu-latest
```

Comment:
Do we have that because releases have a different version number for example, or something of this kind? Ultimately I trust you on this, just trying to think of things that would have lead us to have a different workflow for releases 🤷